app/assets/stylesheets/course/announcements.css.scss

```diff
@@ -0,0 +1,25 @@
+// Place all the styles related to the course/announcements controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+@import "variables";
+@import "mixins/future";
+
+.course-announcements {
+  &.index {
+    .announcement {
+      padding-left: 8px;
```

Comment:
I prefer units that scale, @yangshun what units do you recommend? I tend to use em for things involving block elements. px only for images...